client/source/js/modules/model/cost-coverage.html

```diff
@@ -70,8 +67,8 @@
               </p>
               <p>
                 3. [Optional] Specify scale-up parameters:
-                <input type="number" name="scaleUpParameter" min="1" max="10" step="0.1" class="txbox __inline __xl" placeholder="e.g.{{defaultScaleUpParameter}}"  ng-model="scaleUpParameter" ng-change="updateCurves()">
-                <error-messages for="scaleUpParameter" rules="{min:1, max:10}"></error-messages>
```

Comment:
@robynstuart hmmm the graphs look totally flat with such scale-up parameter, and before, I noticed that they looked better when this parameter was >1. I see that it's being sent correctly though (as ccparams[3])